tsl/test/expected/dist_hypertable-12.out

```diff
@@ -239,91 +239,43 @@ UPDATE disttable SET temp = 3.7 WHERE device = 1;
    Update on public.disttable
    Foreign Update on _timescaledb_internal._dist_hyper_1_1_chunk
      Remote SQL: UPDATE _timescaledb_internal._dist_hyper_1_1_chunk SET temp = $2 WHERE ctid = $1
-   Foreign Update on _timescaledb_internal._dist_hyper_1_2_chunk
-     Remote SQL: UPDATE _timescaledb_internal._dist_hyper_1_2_chunk SET temp = $2 WHERE ctid = $1
-   Foreign Update on _timescaledb_internal._dist_hyper_1_3_chunk
-     Remote SQL: UPDATE _timescaledb_internal._dist_hyper_1_3_chunk SET temp = $2 WHERE ctid = $1
    Foreign Update on _timescaledb_internal._dist_hyper_1_4_chunk
      Remote SQL: UPDATE _timescaledb_internal._dist_hyper_1_4_chunk SET temp = $2 WHERE ctid = $1
-   Foreign Update on _timescaledb_internal._dist_hyper_1_5_chunk
-     Remote SQL: UPDATE _timescaledb_internal._dist_hyper_1_5_chunk SET temp = $2 WHERE ctid = $1
-   Foreign Update on _timescaledb_internal._dist_hyper_1_6_chunk
-     Remote SQL: UPDATE _timescaledb_internal._dist_hyper_1_6_chunk SET temp = $2 WHERE ctid = $1
    ->  Seq Scan on public.disttable
          Output: disttable."time", disttable.device, NULL::integer, '3.7'::double precision, disttable.ctid
-         Filter: (disttable.device = 1)
+         Filter: ((disttable.device = 1) AND (_timescaledb_internal.get_partition_hash(disttable.device) = 242423622))
```

Comment:
Do you want to delete them back as well as you did with constraints for `now()`?

Reply:
yes they shouldnt end up in the explain output, will fix this

Reply:
we could make the constraint removal a guc but i'm somewhat hesitant to add a GUC like that